core-processor/Cargo.toml

```diff
@@ -11,13 +11,10 @@ gear-core-errors = { path = "../core-errors", features = ["codec"] }
 gear-backend-common = { path = "../core-backend/common" }
 
 codec = { package = "parity-scale-codec", version = "3.1.2", features = ["derive"], default-features = false }
-scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
+scale-info = { version = "2.2.0", default-features = false, features = ["derive"] }
 log = { version = "0.4.17", default-features = false }
-anyhow = { version = "1.0.65", default-features = false }
 derive_more = "0.99.17"
 
-[dev-dependencies]
-hex = "0.4.3"
-
 [features]
 strict = []
+verbose = []
```

Comment:
Seems that superfluous work for now, so I'd remove it

Reply:
Otherwise need to adapt all scripts etc

Reply:
Ok, I remove it at the moment but need to add it in the future as `log::trace` forces to iterate `new_pages_data` in `executor::get_pages_to_be_updated`.